libs/src/api/user.js

```diff
@@ -142,7 +140,6 @@ class Users extends Base {
 
     const result = await this.contracts.UserFactoryClient.addUser(metadata.handle)
     await this._addUserOperations(result.userId, metadata)
```

Comment:
@raymondjacobson @SidSethi i just noticed there's no this.userStateManager.setCurrentUser call here. do we need one?

Reply:
totally. great catch